### PR TITLE
test(e2e): Add secrets to rdws apprunner suite

### PR DESCRIPTION
**Pull request by yohanthshetty**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/commands/secret-init.en.md [hunk 1]
```diff
@@ -8,9 +8,6 @@ $ copilot secret init
 
 A secret can have different values in each of your existing environments, and is accessible by your services or jobs from the same application and environment.
 
-!!! attention 
-    Secrets are not supported for Request-Driven Web Services.
-
```
Comment (iamhopaul123) [inline site/content/docs/commands/secret-init.en.md:13]: this change seems to happen before. did you work on the latest commit in mainline?

File: e2e/apprunner/copilot/back-end/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#
+# variables:                    # Pass environment variables as key value pairs.
+#   LOG_LEVEL: info
+#
+# tags:                         # Pass tags as key value pairs.
+#   project: project-name
+#
+# secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store and Secrets Manager.
+#   GITHUB_TOKEN: GITHUB_TOKEN  # The key is the name of the environment variable, the value is the name of an SSM parameter.
+
```
Comment (iamhopaul123) [inline e2e/apprunner/copilot/back-end/manifest.yml:51]: why do we create the manifest file without the secrets enabled? if it is not enabled how do we know if this feature works from e2e?

File: site/content/docs/commands/secret-init.en.md (hunk 1 above)
Comment (yohanthshetty) [reply site/content/docs/commands/secret-init.en.md:13]: This was already pushed and merged. I committed on the same branch again. Should I send out a new PR?

File: e2e/apprunner/copilot/back-end/manifest.yml (hunk 2 above)
Comment (yohanthshetty) [reply e2e/apprunner/copilot/back-end/manifest.yml:51]: I might have missed it. I'll send out an entirely new PR.

**Pull request closed without merging**